subsys/bluetooth/controller/ll_sw/ti/hal/cc13xx_cc26xx/ll_irqs.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2019 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <inc/hw_ints.h>
+
+/* for HAL_SWI_RADIO_IRQ */
+#define LL_SWI4_IRQn (INT_I2C_IRQ - 16)
```

Comment:
This needs to change. @vanti can you suggest a better IRQ to hijack?

Reply:
I'd suggest you use the SwiP module from hal_ti for SWI functionality, instead of grabbing onto random interrupt lines for the same. SwiP automatically multiplexes the SWI instances to share a single interrupt line. In lll_init(), instead of calling IRQ_CONNECT/irq_enable etc. to set up an IRQ, call SwiP_construct() to create a SWI that would invoke swi_lll_cc13xx_cc26xx_isr(), and implement hal_swi_lll_pend() with SwiP_post(). You can probably use the same trick for other SWI IRQs as well.

Reply:
Sounds good. I'll work on that today.

Reply:
Covered under the task list.